+++
title = "Exploring deep phylogenetic incongruence of the COM clade in Rosidae: Phylogenomics approach [PhD dissertation]"
date = 2014-05-28T00:00:00
draft = false

# Authors. Comma separated list, e.g. `["Bob Smith", "David Jones"]`.
authors = ["**Miao Sun**"]

# Publication type.
# Legend:
# 0 = Uncategorized
# 1 = Conference paper
# 2 = Journal article
# 3 = Manuscript
# 4 = Report
# 5 = Book
# 6 = Book section
publication_types = ["0"]

# Publication name and optional abbreviated version.
publication = "In *Institute of Botany, Chinese Acandemy of Sciences*"
publication_short = "In *IBCAS*"

# Abstract and optional shortened version.
abstract = "Analysis of large data sets can help resolve difficult nodes in the tree of life and also concomitantly reveal complex evolutionary histories, including instances of lateral gene transfer, hybridization, or incomplete lineage sorting. The placement of the **Celastrales-Oxalidales-Malpighiales (COM) clade** within the large Rosidae clade remains one of the most difficult deep-level phylogenetic questions in angiosperms, with previous analyses placing it with either _Fabidae_ (suggested by chloroplast genes) or _Malvidae_ (suggested by mitochondrial and nuclear genes, as well as morphological data). To elucidate the underlying cause of this phylogenetic discordance, we assembled taxonomically comparable multi-gene matrices of chloroplast, mitochondrial, and nuclear sequences (82 taxa for 78 chloroplast genes, 79 taxa for 4 mitochondrial genes, and 92 taxa for 5 nuclear genes), as well as large single- and multi-copy nuclear gene data sets (8,445 single-copy ortholog sets and 3,748 multi-copy nuclear gene families). Analyses of multi-gene data sets demonstrate incongruence between the chloroplast and both nuclear and mitochondrial data sets, and the results are robust to various character-coding and data-exclusion treatments, not due to systematic biases or sampling errors. Analyses of single- and multi-copy nuclear genes indicate that most loci support the placement of COM with _Malvidae_, with a notable number of genes supporting COM with _Fabidae_, and almost no support for COM outside a clade of _Malvidae_ and _Fabidae_. The proportion of genes supporting each hypothesis suggests that the phylogenetic incongruence is not due to incomplete lineage sorting, remaining ancient introgressive hybridization as a plausible explanation for the conflict among genes. In summary, our analyses demonstrate that the placement of COM clade with Malvidae better reflects organismal phylogeny, the conflicting phylogenetic placements for COM clade might be caused by ancient hybridization and chloroplast transmission occurred during the early and rapid radiation of _Rosidae_, and consequently resulted in conflict between chloroplast and mitochondrial gene trees, as well as a mixture of two underlying signals derived from its ancestral parents in the nuclear genome. Although greater taxon and nuclear genome sampling are necessary to evaluate such hypothesis fully, our study provides an example for examination of other deep nodes of the tree of life where conflict occurs among data sets from different subcellular compartments, and also emphasizes the importance of genomic data sets for revealing deep incongruence and potentially complex patterns of evolution in organismal phylogeny."
abstract_short = ""

# Is this a selected publication? (true/false)
selected = false

# Projects (optional).
#   Associate this publication with one or more of your projects.
#   Simply enter your project's folder or file name without extension.
#   E.g. `projects = ["deep-learning"]` references 
#   `content/project/deep-learning/index.md`.
#   Otherwise, set `projects = []`.
projects = []

# Slides (optional).
#   Associate this publication with Markdown slides.
#   Simply enter your slide deck's filename without extension.
#   E.g. `slides = "example-slides"` references 
#   `content/slides/example-slides.md`.
#   Otherwise, set `slides = ""`.
# slides = "example-slides"

# Tags (optional).
#   Set `tags = []` for no tags, or use the form `tags = ["A Tag", "Another Tag"]` for one or more tags.
tags = ["Hybridization", "Incomplete Lineage Sorting", "Phylogeny", "Incongruence", "COM", "Phylogenomics"]

# Links (optional).
url_pdf = "files/Miao_Dissertation.pdf"
# url_preprint = ""
# url_code = ""
# url_dataset = ""
# url_project = ""
url_slides = "Miao20140528.pdf"
# url_video = ""
# url_poster = ""
# url_source = ""

# Custom links (optional).
#   Uncomment line below to enable. For multiple links, use the form `[{...}, {...}, {...}]`.
# url_custom = [{name = "Custom Link", url = "http://example.org"}]

# Digital Object Identifier (DOI)
doi = ""

# Does this page contain LaTeX math? (true/false)
math = false

# Featured image
# To use, add an image named `featured.jpg/png` to your page's folder. 
[image]
  # Caption (optional)
  caption = "Miao's Dissertation: Fig. 4-1"

  # Focal point (optional)
  # Options: Smart, Center, TopLeft, Top, TopRight, Left, Right, BottomLeft, Bottom, BottomRight
  focal_point = "Smart"
+++
