+++
title = "APPS: A new resource for the development of SSR markers: Millions of loci from a thousand plant transcriptomes"
date = 2016-01-01T00:00:00
draft = false

# Authors. Comma separated list, e.g. `["Bob Smith", "David Jones"]`.
authors = ["Richard GJ Hodel", "Matthew A Gitzendanner", "Charlotte C Germain-Aubrey", "Xiaoxian Liu", "Andrew A Crowl", "**_Miao Sun_**", "Jacob B Landis", "M Claudia Segovia-Salcedo", "Norman A Douglas", "Shichao Chen", "others"]

# Publication type.
# Legend:
# 0 = Uncategorized
# 1 = Conference paper
# 2 = Journal article
# 3 = Manuscript
# 4 = Report
# 5 = Book
# 6 = Book section
publication_types = ["2"]

# Publication name and optional abbreviated version.
publication = "In *Applications in Plant Sciences*"
publication_short = "In *APPS*"

# Abstract and optional shortened version.
abstract = "**Premise of the study:** The One Thousand Plant Transcriptomes Project (1KP, 1000+ assembled plant transcriptomes) provides an enormous resource for developing microsatellite loci across the plant tree of life. We developed loci from these transcriptomes and tested their utility. **Methods and Results:** Using software packages and custom scripts, we identified microsatellite loci in 1KP transcriptomes. We assessed the potential for cross‐amplification and whether loci were biased toward exons, as compared to markers derived from genomic DNA. We characterized over 5.7 million simple sequence repeat (SSR) loci from 1334 plant transcriptomes. Eighteen percent of loci substantially overlapped with open reading frames (ORFs), and electronic PCR revealed that over half the loci would amplify successfully in conspecific taxa. Transcriptomic SSRs were approximately three times more likely to map to translated regions than genomic SSRs. **Conclusions:** We believe microsatellites still have a place in the genomic age—they remain effective and cost‐efficient markers. The loci presented here are a valuable resource for researchers."
abstract_short = ""

# Is this a selected publication? (true/false)
selected = false

# Projects (optional).
#   Associate this publication with one or more of your projects.
#   Simply enter your project's folder or file name without extension.
#   E.g. `projects = ["deep-learning"]` references 
#   `content/project/deep-learning/index.md`.
#   Otherwise, set `projects = []`.
projects = []

# Slides (optional).
#   Associate this publication with Markdown slides.
#   Simply enter your slide deck's filename without extension.
#   E.g. `slides = "example-slides"` references 
#   `content/slides/example-slides.md`.
#   Otherwise, set `slides = ""`.
# slides = "example-slides"

# Tags (optional).
#   Set `tags = []` for no tags, or use the form `tags = ["A Tag", "Another Tag"]` for one or more tags.
tags = ["1KP", "SSR", "Phylogeny", "Microsatellites", "Plant Transcriptomes"]

# Links (optional).
url_pdf = "https://doi.org/10.3732/apps.1600024"
# url_preprint = ""
# url_code = ""
# url_dataset = ""
# url_project = ""
# url_slides = ""
# url_video = ""
# url_poster = ""
url_source = "https://bsapubs.onlinelibrary.wiley.com/doi/full/10.3732/apps.1600024"

# Custom links (optional).
#   Uncomment line below to enable. For multiple links, use the form `[{...}, {...}, {...}]`.
# url_custom = [{name = "Custom Link", url = "http://example.org"}]

# Digital Object Identifier (DOI)
doi = "10.3732/apps.1600024"

# Does this page contain LaTeX math? (true/false)
math = false

# Featured image
# To use, add an image named `featured.jpg/png` to your page's folder. 
[image]
  # Caption (optional)
  caption = ""

  # Focal point (optional)
  # Options: Smart, Center, TopLeft, Top, TopRight, Left, Right, BottomLeft, Bottom, BottomRight
  focal_point = "Smart"
+++
