+++
title = "Demonstrating gene tree conflict with Phyparts Piecharts"
subtitle = ""

date = 2019-10-27T00:00:00
draft = false

# Authors. Comma separated list, e.g. `["Bob Smith", "David Jones"]`.
authors = ["Miao Sun"]

tags = ["Dimensions", "Incongruence", "Phyparts Piecharts", "ASTRAL", "Phylogeny", "Genomics", "Phylogenomics"]
summary = "Explained the importance of understanding the underlying conflict among gene trees. List all the steps required to reach the final stage of data visualization. Along with each step, necesary methods (software, an scripts) and data sets are also deatailed."

# Projects (optional).
#   Associate this post with one or more of your projects.
#   Simply enter your project's folder or file name without extension.
#   E.g. `projects = ["deep-learning"]` references 
#   `content/project/deep-learning/index.md`.
#   Otherwise, set `projects = []`.
 projects = ["tree_of_life", "dimensions"]

# Featured image
# To use, add an image named `featured.jpg/png` to your project's folder. 
[image]
  # Caption (optional)
  caption = "PhypartsPiecharts for Sassafras"

  # Focal point (optional)
  # Options: Smart, Center, TopLeft, Top, TopRight, Left, Right, BottomLeft, Bottom, BottomRight
  focal_point = ""

  # Show image only in page previews?
  preview_only = true

+++
_We have marched into genome era already, especially using biparental inherited nuclear genome data to examine the framework of tree of life established by plastid data. However, not all gene trees tell one same story, even the most homogeneous genes from plastid organelle. Not to mention nuclear genes may have more than one copy, and other potential biological processes (e.g., hybridization, incomplete lineage sorting (ILS), or horizontal gene transfer, etc); also see [Sun et al. (2015)]({{< ref "/publication/sun2015com/index.md" >}}). Hence, [Smith et al. (2015)](https://bmcevolbiol.biomedcentral.com/articles/10.1186/s12862-015-0423-0) did a great job in summarizing current situation, and setting examples of visualization of concordance information in animals and plants genomic phylogenies. They also developed an open source java software to do this job --- [phyparts](https://bitbucket.org/blackrim/phyparts)._  

  In this post I will use _Sassafras_ 322 gene trees as an example to show you how to use [ASTRAL-III](https://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-018-2129-y) to estimate a species tree, and [phyparts](https://bitbucket.org/blackrim/phyparts) to summarize the conflict and concordance information of those individual homologous gene regions, and finally using [PhyParts PieCharts](https://github.com/mossmatters/MJPythonNotebooks/blob/master/PhyParts_PieCharts.ipynb) to visualize the Phyparts Output. These data was generated by Target Enrichment method using [Universal Probe Set for Targeted Sequencing of 353 Nuclear Genes](https://academic.oup.com/sysbio/article/68/4/594/5237557) under [Dimension project]({{< ref "/project/dimensions/index.md" >}}). My other relevant workingflow and scripts are available [here](https://github.com/Cactusolo/IESHTSTE).

*Note: [Matt Johnson](https://github.com/mossmatters/MJPythonNotebooks/blob/master/PhyParts_PieCharts.ipynb) has a great tutorial of how to run his script using Jupyter Notebook. The interpretation of PieCharts is well explained as well.*

My instructions here focus on the overall procedures from gene trees all the way down to the Pie chart. Hopefully, I'm able to integrate all the steps together into one pipeline.


## General steps  

_Here we need to assess how a number of gene trees how they agree with ours pecies tree, and displaying this discordance and agreeent information with satisfying visualization._ 

### 1. Building phylogeny, so that we have all gene trees  
  - My data was generated from 353 target enrichment method
  - I used 322 gene trees from _Sassafras_ samples as example
  - Note:  
    + All trees were rooted by either [phyx](https://github.com/FePhyFoFum/phyx) or [The Newick Utilities](https://github.com/tjunier/newick_utils/wiki), or [DendroPy](https://pypi.org/project/DendroPy/)  
    + Though [ASTRAL](https://github.com/smirarab/ASTRAL/blob/master/astral-tutorial.md#running-with-unresolved-gene-trees) is able to take any gene trees rooted or unrooted, however, for downstream Phyparts performance, rooted trees are preferred. Such so all gene trees and species trees are towards to the same root direction (See [Matt Johnson's post](https://github.com/mossmatters/MJPythonNotebooks/blob/master/PhyParts_PieCharts.ipynb)).
    + As quote in [ASTRAL tutorial](https://github.com/smirarab/ASTRAL/blob/master/astral-tutorial.md#running-with-unresolved-gene-trees):   
    _"Importantly, we will reroot the tree at the correct node, which is always necessary, since the rooting of the ASTRAL trees is arbitrary and meaningless."_  
    
    **Therefore, rooting all the trees (species tree and gene trees) is preferred for Phyparts Piecharts. Sometimes, when you have hundreds of gene trees, however, not all outgroups presented in your gene tree; in these case, I recommend either use `pxrr` function from [phyx](https://github.com/FePhyFoFum/phyx), or [MAD (root the tree by Minimal Ancestor Deviation)](https://www.nature.com/articles/s41559-017-0193); the latter works well with those tree without outgroups at all.; for details see my post [here]({{< ref "/post/2020-11-26-check-the-performance-of-a-rooting-method-using-minimal-ancestor-deviation-mad.en.Rmd" >}})**  
  
### 2. Species tree estimation  

  - Species tree was estimated by [ASTRAL-III](https://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-018-2129-y), and see tutorials [here](https://github.com/smirarab/ASTRAL/blob/master/astral-tutorial.md#running-on-the-sample-mammalian-dataset)
  - **Note:** Collapse gene tree nodes with BS support less than certain value (saying 10%; see cmd below), will help to improve accuracy; sometime increasing the threshold for collapse may yield better results.   
  There are many ways to do this: [phyx](https://github.com/FePhyFoFum/phyx) or [The Newick Utilities](https://github.com/tjunier/newick_utils/wiki)  
    + using The Newick Utilities as example from [ASTRAL-III tutorial](https://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-018-2129-y) 
      e.g., `nw_ed  1KP-genetrees.tre 'i & b<=10' o > 1KP-genetrees-BS10.tre`  
      
  - ASTRAL **CMD**:  
    `java -jar astral.5.6.3.jar -i collapse_genetrees.tre -o output_species_tree.tre 2> running.log`  
    
### 3. Statistic information about conflict, concordance, or even gene duplications 

  - You need to run  [phyparts](https://bitbucket.org/blackrim/phyparts/src/master/) with instruction on the repo webpage.
  - Phyparts "conflict" option **CMD**:  
    `java -jar target/phyparts-0.0.1-SNAPSHOT-jar-with-dependencies.jar -a 1 -v -d gene_trees -m ASTRAL_species_tree -o output_name`  
    
  _Note:_ run cmd below, you'll able to see all the Phyparts options (see snapshot below):  
      `java -jar target/phyparts-0.0.1-SNAPSHOT-jar-with-dependencies.jar`
  
  {{< figure src="phyparts_option.jpg" title="" numbered="true" >}}
### 4. Mapping these information on species tree
  - You need **PhyParts PieCharts** [python script](https://github.com/mossmatters/phyloscripts/tree/master/phypartspiecharts) and tutorial from [Matt Johnson](https://github.com/mossmatters/MJPythonNotebooks/blob/master/PhyParts_PieCharts.ipynb)  
  
_Note: Please read the tutorial, **Python >2.7 and ETE3** with the graphical options need to be installed before run the python script_  
  
  - how to run script:  
    `python3 phypartspiecharts.py species_tree output_name gene_number`  
    
  *Note: `output_name` must be the same as you names at the `phyparts (# 3)` step*  
    
### 5. Visualize the gene tree support and conflic statics with Piecharts##
  - the output format of PhyParts PieCharts is in `svg`, then you need to Inkscape or AI to convert it as `pdf`. You also can refine the overall layout of that plot.  
  
  - Note: make sure Python3, ETE3, and [X server](https://kovyrin.net/2007/10/01/how-to-run-gui-programs-on-a-server-without-any-monitor/) installed.
  
  As my plot below:  

{{< figure src="featured.jpg" title="" numbered="true" >}}

Last updated: Mon Dec  7 2020



