+++
title = "JSE: Tree of life for the genera of Chinese vascular plants"
date = 2016-01-01T00:00:00
draft = false

# Authors. Comma separated list, e.g. `["Bob Smith", "David Jones"]`.
authors = ["Zhi-Duan Chen", "Tuo Yang", "Li Lin", "Li-Min Lu", "Hong-Lei Li", "**_Miao Sun_**", "Bing Liu", "Min Chen", "Yan-Ting Niu", "Jian-Fei Ye", "others"]

# Publication type.
# Legend:
# 0 = Uncategorized
# 1 = Conference paper
# 2 = Journal article
# 3 = Manuscript
# 4 = Report
# 5 = Book
# 6 = Book section
publication_types = ["2"]

# Publication name and optional abbreviated version.
publication = "In *Journal of Systematics and Evolution*"
publication_short = "In *JSE*"

# Abstract and optional shortened version.
abstract = "We reconstructed a phylogenetic tree of Chinese vascular plants (Tracheophyta) using sequences of the chloroplast genes _atpB_, _matK_, _ndhF_, and _rbcL_ and mitochondrial _matR_. We produced a matrix comprising 6,098 species and including 13,695 DNA sequences, of which 1,803 were newly generated. Our taxonomic sampling spanned 3,114 genera representing 323 families of Chinese vascular plants, covering more than 93% of all genera known from China. The comprehensive large phylogeny supports most relationships among and within families recognized by recent molecular phylogenetic studies for lycophytes, ferns (monilophytes), gymnosperms, and angiosperms. For angiosperms, most families in Angiosperm Phylogeny Group IV are supported as monophyletic, except for a paraphyletic Dipterocarpaceae and Santalaceae. The infrafamilial relationships of several large families and monophyly of some large genera are well supported by our dense taxonomic sampling. Our results showed that two species of _Eberhardtia_ are sister to a clade formed by all other taxa of Sapotaceae, except _Sarcosperma_. We have made our phylogeny of Chinese vascular plants publically available for the creation of subtrees via [SoTree](http://www.darwintree.cn/flora/index.shtml), an automated phylogeny assembly tool for ecologists."
abstract_short = ""

# Is this a selected publication? (true/false)
selected = false

# Projects (optional).
#   Associate this publication with one or more of your projects.
#   Simply enter your project's folder or file name without extension.
#   E.g. `projects = ["deep-learning"]` references 
#   `content/project/deep-learning/index.md`.
#   Otherwise, set `projects = []`.
projects = ["tree_of_life"]

# Slides (optional).
#   Associate this publication with Markdown slides.
#   Simply enter your slide deck's filename without extension.
#   E.g. `slides = "example-slides"` references 
#   `content/slides/example-slides.md`.
#   Otherwise, set `slides = ""`.
# slides = "example-slides"

# Tags (optional).
#   Set `tags = []` for no tags, or use the form `tags = ["A Tag", "Another Tag"]` for one or more tags.
tags = ["Chinese vascular plants", "Tree of Life", "Phylogeny", "SoTree"]

# Links (optional).
url_pdf = "https://doi.org/10.1111/jse.12219"
# url_preprint = ""
# url_code = ""
# url_dataset = ""
# url_project = ""
# url_slides = ""
# url_video = ""
# url_poster = ""
# url_source = ""

# Custom links (optional).
#   Uncomment line below to enable. For multiple links, use the form `[{...}, {...}, {...}]`.
# url_custom = [{name = "Custom Link", url = "http://example.org"}]

# Digital Object Identifier (DOI)
doi = "10.1111/jse.12219"

# Does this page contain LaTeX math? (true/false)
math = false

# Featured image
# To use, add an image named `featured.jpg/png` to your page's folder. 
[image]
  # Caption (optional)
  caption = "Chen et al. (2016): Fig. 1"

  # Focal point (optional)
  # Options: Smart, Center, TopLeft, Top, TopRight, Left, Right, BottomLeft, Bottom, BottomRight
  focal_point = "Smart"
+++
