+++
title = "Genome: Identification of nuclear low-copy genes and their phylogenetic utility in rosids"
date = 2014-01-01T00:00:00
draft = false

# Authors. Comma separated list, e.g. `["Bob Smith", "David Jones"]`.
authors = ["Baohua Wang", "Yan Zhang", "Peipei Wei", "**_Miao Sun_**", "Xiaofei Ma", "Xinyu Zhu"]

# Publication type.
# Legend:
# 0 = Uncategorized
# 1 = Conference paper
# 2 = Journal article
# 3 = Manuscript
# 4 = Report
# 5 = Book
# 6 = Book section
publication_types = ["2"]

# Publication name and optional abbreviated version.
publication = "In *Genome*"
publication_short = "In *Genome*"

# Abstract and optional shortened version.
abstract = "By far, the interordinal relationships in rosids remain poorly resolved. Previous studies based on chloroplast, mitochondrial, and nuclear DNA has produced conflicting phylogenetic resolutions that has become a widely concerned problem in recent phylogenetic studies. Here, a total of 96 single-copy nuclear gene loci were identified from the KOG (eukaryotic orthologous groups) database, most of which were first used for phylogenetic analysis of angiosperms. The orthologous sequence datasets from completely sequenced genomes of rosids were assembled for the resolution of the position of the **COM (Celastrales–Oxalidales–Malpighiales) clade** in rosids. Our analysis revealed strong and consistent support for CM topology (the COM clade as sister to the malvids). Our results will contribute to further exploring the underlying cause of conflict between chloroplast, mitochondrial, and nuclear data. In addition, our study identified a few novel nuclear molecular markers with potential to investigate the deep phylogenetic relationship of plants or other eukaryotic taxonomical groups."
abstract_short = ""

# Is this a selected publication? (true/false)
selected = false

# Projects (optional).
#   Associate this publication with one or more of your projects.
#   Simply enter your project's folder or file name without extension.
#   E.g. `projects = ["deep-learning"]` references 
#   `content/project/deep-learning/index.md`.
#   Otherwise, set `projects = []`.
projects = []

# Slides (optional).
#   Associate this publication with Markdown slides.
#   Simply enter your slide deck's filename without extension.
#   E.g. `slides = "example-slides"` references 
#   `content/slides/example-slides.md`.
#   Otherwise, set `slides = ""`.
# slides = "example-slides"

# Tags (optional).
#   Set `tags = []` for no tags, or use the form `tags = ["A Tag", "Another Tag"]` for one or more tags.
tags = ["rosids", "COM", "Phylogeny", "Incongruence", "Nuclear Genes"]

# Links (optional).
url_pdf = "https://www.researchgate.net/publication/272406494_Identification_of_nuclear_low-copy_genes_and_their_phylogenetic_utility_in_rosids"
# url_preprint = ""
# url_code = ""
# url_dataset = ""
# url_project = ""
# url_slides = ""
# url_video = ""
# url_poster = ""
url_source = "http://www.nrcresearchpress.com/doi/abs/10.1139/gen-2014-0138#.XA0ybBM3kWo"

# Custom links (optional).
#   Uncomment line below to enable. For multiple links, use the form `[{...}, {...}, {...}]`.
# url_custom = [{name = "Custom Link", url = "http://example.org"}]

# Digital Object Identifier (DOI)
doi = "10.1139/gen-2014-0138"

# Does this page contain LaTeX math? (true/false)
math = false

# Featured image
# To use, add an image named `featured.jpg/png` to your page's folder. 
[image]
  # Caption (optional)
  caption = "Wang et al. (2014): Fig. 1"

  # Focal point (optional)
  # Options: Smart, Center, TopLeft, Top, TopRight, Left, Right, BottomLeft, Bottom, BottomRight
  focal_point = "Smart"
+++
